AmplifyPlugins/Auth/AWSAuthPlugin/Models/AWSAuthUser.swift

```diff
@@ -9,8 +9,15 @@ import Amplify
 
 public struct AWSAuthUser: AuthUser {
 
+    /// The username for the logged in user
+    ///
+    /// Value maps to the username of a user in AWS Cognito User Pool. This value is set by AWS Cognito and not by the
+    /// user and does not always map with the username used to signIn.
```

Comment:
As noted offline, we need to figure out this behavior.

Reply:
Keeping this for now as discussed